Plots/Station/NCL_station_1.py

```diff
@@ -0,0 +1,130 @@
+"""
+NCL_station_1.py
+================
+This script illustrates the following concepts:
+   - Using pandas package to read in ascii file with several columns of data
+   - Using tricontour and tricontourf function from matplotlib package to contour one-dimensional X, Y, Z data
+   - Drawing lat/lon locations as filled dots
+   - Controlling which contour lines get drawn
+   - Using alpha parameter to emphasize or subdue overlain features
+   - Using a different color scheme to follow `best practices <https://geocat-examples.readthedocs.io/en/latest/gallery/Colors/CB_Temperature.html#sphx-glr-gallery-colors-cb-temperature-py`_ for visualizations
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/station_1.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/station_1_lg.png
+"""
+
+###################################################
+# Import packages:
+
+import numpy as np
+import pandas as pd
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+from matplotlib import pyplot as plt
+
+import geocat.datafiles as gdf
+from geocat.viz import util as gvutil
+
+###################################################
+# Generate data:
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = pd.read_csv(gdf.get('ascii_files/pw.dat'), delimiter='\\s+')
+
+# Extract columns
+pwv = ds.PW
+pwv_lat1d = ds.LAT
+pwv_lon1d = ds.LON
+
+###################################################
+# Plot
+
+# Generate figure (set its size (width, height) in inches)
+fig = plt.figure(figsize=(12, 10))
+
+# Generste axes
+ax = plt.axes(projection=ccrs.PlateCarree())
+
+# Specify contour and contourf levels
+clevels = np.arange(25, 51, 5)
+flevels = np.arange(16, 51, 1)
+
+# Specify colormap and reverse it
```

Comment:
```suggestion
# Specify colormap
```
The reversal doesn't actually happen here, I thought this comment might be a little confusing